pkgs/by-name/av/avdump3/package.nix

```
@@ -0,0 +1,43 @@
+{ lib, stdenv, writeScriptBin, fetchzip, dotnet-runtime, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "avdump3";
+  version = "8293_stable";
+
+  src = fetchzip {
+    url = "https://cdn.anidb.net/client/avdump3/avdump3_8293_stable.zip";
+    hash = "sha256-H9Sn3I4S9CmymKIMHVagDy+7svHs285S3EJgYQo+ks0=";
+    stripRoot = false;
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/${pname} $out/bin
+    mv * $out/share/${pname}
+    cat > $out/bin/${pname} <<EOF
+    #!/bin/sh
+    export LD_LIBRARY_PATH="${lib.makeLibraryPath [ zlib ]}:\$LD_LIBRARY_PATH"
+    exec ${dotnet-runtime}/bin/dotnet $out/share/${pname}/AVDump3CL.dll "\$@"
+    EOF
+    chmod +x $out/bin/${pname}
+  '';
+
+  dontPatchELF = true;
+
+  meta = with lib; {
```

```suggestion
  meta = {
```

Better to avoid `with lib`